README.md

```diff
@@ -65,7 +65,7 @@
 
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] For the transaction page, Please add the correct margin-top value to the page title by decreasing it, add the correct card styles for the transactions added to the page, and finally add the correct placement for the `add transaction` button to be at the bottom of the page to match the given design.

https://github.com/Samuelwanza/financier/blob/f30decf7b5075dcd5ee2651363a40aefc07b633d/app/views/payments/index.html.erb#L1-L10

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Samuelwanza/financier/assets/68796232/c54d4a0e-16ba-4459-98cb-244a567dff0d) | ![image](https://github.com/Samuelwanza/financier/assets/68796232/a1c41323-b0f3-4c1d-a3df-15a398524d7b)  |